InfiniteRecharge/src/test/java/frc/robot/subsystem/scoring/intake/IntakeSubsystemTest.java

```diff
@@ -58,14 +58,14 @@
     public void beforeTest() throws Exception {
         super.beforeTest();
         config = ConfigChooser.getConfig();
-        config.intake.intakePivotEnabled = true;
+        IntakeConfig.intakePivotEnabled = true;
 
         // when our subsystem is initialized, it will create a motor and a solenoid. We
         // don't want
         // actual motors and solenoids to be created, we want mock ones. Make sure we
         // return our mocked
         // instances instead of new real instances.
-        whenNew(WPI_TalonSRX.class).withArguments(eq(config.intake.intake.id)).thenReturn(motor);
+        whenNew(WPI_TalonSRX.class).withArguments(eq(Config.INTAKE_MOTOR_ID)).thenReturn(motor);
```

Comment:
Even though we left the constants in Config.java, we should use the configured config.intake.intake.id value because it will change from robot to robot.

Reply:
`Config.INTAKE_MOTOR_ID` is like the default motor id for Shawty, but it will be different on different bots.